sources/FASLOAD

```diff
@@ -1,14 +1,14 @@
 (DEFINE-FILE-INFO READTABLE "XCL" PACKAGE "FASL")
-(IL:FILECREATED "17-Apr-2018 07:55:20" 
-IL:|{DSK}<Users>kaplan>Local>medley3.5>lispcore>sources>FASLOAD.;2| 35249  
+(IL:FILECREATED "10-Jun-2021 18:26:43" 
+IL:|{DSK}<Users>kaplan>Local>medley3.5>git-medley>sources>FASLOAD.;8| 35584  
 
-      IL:|changes| IL:|to:|  (IL:FNS CONVERT-FASL-DATE)
```

Comment:
This is really illegible IL:|changes| IL:|to:|

Reply:
Happens any time the FILECREATED expressions are written/read with a readtable other than an IL based one.  IL:|previous| IL:|date|, etc.

Reply:
a separate issue, but I think we should change the FILECREATED format to use strings or keywords

   "changes to" IL:READ "previous" "12-DEC-2021"

or

   :changes "IL:READ" :previous-date "12-Dec-2021"

so it isn't so ugly. This is in support of "Modernization" and "better support of Common Lisp".